sources/academy/webscraping/web_scraping_for_beginners/crawling/pro_scraping.md

```diff
@@ -5,204 +5,227 @@ sidebar_position: 7
+3. **Faster and cheaper scrapers** because Crawlee automatically scales based on system resources, and we optimize its performance regularly.
+4. **More robust scrapers**. Annoying details like retries, proxy management, error handling, and result storage are all handled out-of-the-box by Crawlee.
+5. **Helpful community**. You can [join our Discord](https://discord.gg/qkMS6pU4cF) or talk to us [on GitHub](https://github.com/apify/crawlee). We're almost always there to talk about scraping and programming in general.
 
-> If you're still not convinced, [read this story](https://apify.com/success-stories/daltix-analytics-scrapy-python-to-apify) how a data analytics company saved 90% of scraping costs by switching from Scrapy (a scraping library for Python) to Crawlee. To be honest, it shocked even us! 💪
+:::tip
 
-The Crawlee library factors away and manages the hard parts of the scraping/automation development under the hood, such as:
+If you're still not convinced, [read this story](https://apify.com/success-stories/daltix-analytics-scrapy-python-to-apify) how a data analytics company saved 90% of scraping costs by switching from Scrapy (a scraping library for Python) to Crawlee. We were pretty surprised ourselves, to be honest.
 
-- Autoscaling
+:::
+
+Crawlee factors away and manages the dull and repetitive parts of web scraper development under the hood, such as:
+
+- Auto-scaling
 - Request concurrency
 - Queueing requests
 - Data storage
 - Using and rotating [proxies](../../anti_scraping/mitigation/proxies.md)
 - Puppeteer/Playwright setup overhead
-- Plus much more!
-
-As a developer, your goal is to take Crawlee, which is quite generic on its own, and write website-specific or use-case-specific [actors](../../../platform/getting_started/actors.md).
+- [See all the features](https://crawlee.dev/docs/introduction)
 
 Crawlee and its resources can be found in various different places:
 
 1. [Official Crawlee documentation](https://crawlee.dev/)
-2. [Crawlee Github repository (source code, issues)](https://github.com/apify/crawlee)
+2. [Crawlee GitHub repository (source code, issues)](https://github.com/apify/crawlee)
 3. [Crawlee on NPM](https://www.npmjs.com/package/crawlee)
 
 ## Install Crawlee {#crawlee-installation}
 
-To use Crawlee, we have to install it from NPM. Let's add it to our project from the previous lessons.
+To use Crawlee, we have to install it from NPM. Let's add it to our project from the previous lessons by executing this command in your project's folder.
 
 ```shell
-npm install crawlee@latest
+npm install crawlee
 ```
 
-In a new file we'll call **crawlee.js**, add the following code:
+After the installation completes, create a new file called **crawlee.js** and add the following code to it:
 
-```js
-// crawlee.js
+```js title=crawlee.js
```

Comment:
maybe better to find some other name? remember we wanted to support various config formats one day, so `crawlee.js` would be considered as one of the files to check too. but we could maybe use `crawlee.config.js` instead to be safer.